### PR TITLE
feat(!status): add feat status

### PR DESCRIPTION
**Pull request by dmitryshelomanov**
ISSUES CLOSED: #52

### CODE REVIEW

File: src/features/status/index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+const { allowWhiteListChat } = require('../../middlewares/allowed-chat')
+const { adminRequired } = require('../../middlewares/admin-required')
+
+
+async function checkStatus({
+  reply, options, message, deleteMessage,
+}) {
+  try {
+    debug('status check')
+    await deleteMessage(message.messge_id)
```
Comment (sergeysova) [inline src/features/status/index.js:12]: typo?

File: src/features/status/index.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    reply(text.status.check(options.username))
```
Comment (sergeysova) [inline src/features/status/index.js:13]: rename `check` to `chatIsServed`

File: src/text/index.js [hunk 3]
```diff
@@ -62,9 +62,16 @@ const text = {
     ),
   },
 
+  status: {
+    check: (botName) => (
+      `Чат обслуживается ботом ${botName}`
```
Comment (sergeysova) [inline src/text/index.js:67]: maybe add `@` to prefix bot name?

File: src/features/status/index.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  }
+  catch (error) {
+    debug('checkStatus ERROR', error)
+  }
+}
+
+module.exports = function statusFeature(bot) {
+  bot.hears(
+    new RegExp(`^${text.commands.status()}( .*)?`),
```
Comment (sergeysova) [inline src/features/status/index.js:22]: remove `( .*)?` from end.

command don't receive any content after `!статус`

File: src/text/index.js [hunk 5]
```diff
@@ -62,9 +62,16 @@ const text = {
     ),
   },
 
+  status: {
+    check: (botName) => (
```
Comment (sergeysova) [inline src/text/index.js:66]: always use object as argument in text.

**Review by sergeysova — CHANGES_REQUESTED** [review]

File: src/features/status/index.js (hunk 1 above)
Comment (dmitryshelomanov) [reply src/features/status/index.js:12]: Где ?

Comment (sergeysova) [reply src/features/status/index.js:12]: `messge_id`

**Pull request closed without merging**